src/main/java/com/sugarcrm/candybean/automation/Candybean.java

```diff
@@ -52,7 +54,13 @@
 	/**
 	 * The default configuration file name to instantiate candybean.
 	 */
-	public static final String DEFAULT_CONFIG_FILE = ROOT_DIR + File.separator + "candybean.config";
+	public static final List<String> DEFAULT_CONFIG_FILES = Arrays.asList(
+			ROOT_DIR + File.separator + "candybean.config",
+			ROOT_DIR + File.separator + "src" + File.separator + "test" +
+					File.separator + "resources" + File.separator + "candybean.config",
+			ROOT_DIR + File.separator + "src" + File.separator + "test" +
+					File.separator + "resources" + File.separator + "voodoo.properties"
```

Comment:
Discussed this in person -- Candybean should not contain references to Voodoo for legal reasons.  While there may be some that are leftover from earlier times, they should be removed.  I'm not super comfortable adding this one, but given time constraints I think we can leave it in for now (don't tell Legal).  Please leave a comment noting that it is deprecated and file a new CB ticket for removing this option, blocked by a new VOOD ticket for renaming the config file at the same time.